File: Unit Testing/Calculator/Class1.cs
namespace Calculator
{
    public class Class1
    {
        private int num1, num2;

        public Class1(int num1, int num2)
        {
            Num1 = num1;
            Num2 = num2;
        }

        public int Num1 { get => num1; set => num1 = value; }
        public int Num2 { get => num2; set => num2 = value; }

        public int Add() { return num1 + num2; }
        public int Sub() { return num1 - num2; }
    }
}

File: Unit Testing/Calculation/Class1.vb
Using System;
Using System.Collections.Generic;
Using System.Linq;
Using System.Text;
Using System.Threading.Tasks;

Namespace Calculation
{
    Public Class Calculations
    {
        Private int num1, num2;

        Public Calculations(int num1, int num2)
        {
            Num1 = num1;
            Num2 = num2;
        }

        Public int Num1 { Get => num1; Set => num1 = value; }
        Public int Num2 { Get => num2; Set => num2 = value; }

        Public int Add() { Return num1 + num2; }
        Public int Sub() { Return num1 - num2; }
    }
}
